dnnv/nn/converters/tensorflow.py

```diff
@@ -227,20 +242,24 @@ def visit_Conv(self, operation):
         x_ = operation.x
         if isinstance(x_, Operation):
             x_ = self.visit(x_)
+        w_ = operation.w
+        if isinstance(w_, Operation):
+            w_ = self.visit(w_)
 
         @self._cached
         def conv_func(*inputs):
-            x = _concretize([x_], inputs)
+            x, weights = _concretize([x_, w_], inputs)
             if len(operation.kernel_shape) != 2:
                 raise NotImplementedError(
                     "Non 2d convolutions are not currently supported."
                 )
-            weights = operation.w
+            if not isinstance(weights, np.ndarray):
+                weights = np.array(weights)
             if operation.b is not None:
                 bias = operation.b
             else:
                 bias = np.zeros((weights.shape[0],), dtype=weights.dtype)
-            assert np.all(operation.dilations == 1)
+            assert np.all(operation.group == 1)
```

Comment:
Is this necessary? Could you add support for this parameter, similar to dilations?